src/views/UserSearch/UserSearch.js

```diff
@@ -433,8 +436,24 @@ class UserSearch extends React.Component {
       return ids;
     }, []);
 
+    const requestData = (startDate, endDate, feeFineOwners) => {
+      let requestData = {
```

Comment:
Why function and object have the same name? It looks confusing. Please change function name. 
And objects better declare with `const`.

Reply:
Fixed